### ISSUE TITLE
Incorporating Google Assistant (and other smart devices) into Verigate

### ISSUE THREAD

**Issue by MrStevenNg** [quipsquaff/project2#144]

**Comment by MrStevenNg** [comment]
I'm assuming that our desired focus with this is towards Google Home and Amazon Alexa smart speakers.

**Comment by trizmo** [comment]
exactly. not necessarily high on the priority list. But a nice feature to add- "OK Google, Don't let Ted in thru the gate- make sure that shit is locked"

**Comment by MrStevenNg** [comment]
Nice well if all agree then we'll put this into consideration for future iterations!
